src/calculation/ClauseResultsBuilder.ts

```diff
@@ -875,10 +878,27 @@ export function getSDEValues(measure: fhir4.Measure, statementResults: cql.State
       if (sde.criteria?.expression) {
         const expression = sde.criteria.expression;
         const result = statementResults[expression];
+        if (
+          !(
+            sde.usage &&
+            sde.usage.length > 0 &&
+            sde.usage[0].coding &&
+            sde.usage[0].coding.length > 0 &&
+            isSupplementalDataUsage(sde.usage[0].coding[0].code)
```

Comment:
If we define usage above this line as `const usage = sde.usage?.[0]?.coding?.[0].code;`. We should just be able to replace this whole string of `&&`s with `!isSupplementalDataUsage(usage)`. Or maybe `usage && !isSupplementalDataUsage(usage)` if we want to allow usage to be optional.

Reply:
Oh man- that is so much better. Love it.